### ISSUE TITLE
Readme : X_val y_val optional

### ISSUE THREAD

**Issue by jpramil** [InseeFrLab/torchTextClassifiers#7]
In the Readme : Just precise the validation set is optional when training a model (used by Lighning for training monitoring).
